src/signal.rs

```diff
@@ -1,5 +1,5 @@
 // Copyright 2019 Intel Corporation. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 //
 // Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
This line should be removed too.

Reply:
Line 5 is copied from Amazon license together with line 4. Do you mean we only tell the companies, removing the original license that companies used, and leave one line for declaring final license?

Reply:
not sure whether it makes sense to include multiple "SPDX-License-Identifier" header:)

Reply:
@andreeaflorescu @sameo Do you know the good way to list the companies and licenses that might be different ?

Reply:
@liujing2 I really don't know what is the best approach here. I think it makes sense to send an email to the rust-vmm email list and ask there.

Reply:
The goal of the SPDX id is to provide a single, simple and unified way to expose under which terms a source code file is licensed. Havinf multiple `SPDX-License-Identifier` is confusing and goes against the point of using SPDX ids. So I'd go with a single one.

In that case, Amazon licensed their piece of code under the Apache 2.0 license (exclusively, not dual licensed). So unless Amazon wants to re license (for dual licensing), I think the single SPDX identifier that we should use here is:

```
SPDX-License-Identifier: Apache-2.0
```

Reply:
If the piece of code also includes some BSD-3-Clause code from crosvm, then we should use:

```
SPDX-License-Identifier: (Apache-2.0 AND BSD-3-Clause)
```

Reply:
Thanks for your guidance.  The PR is updated. :)